lib/iris/util.py

```diff
@@ -2008,7 +2117,6 @@ def _strip_metadata_from_dims(cube, dims):
 
     To be used by operations that modify or remove dimensions.
     Note: does nothing to (aux)-coordinates.  Those would be handled explicitly by the calling operation.
-
     """
     reduced_cube = cube.copy()
 
```

Comment:
@ESadek-MO You seem to have missed the following functions:

- [ ] `approx_equal`
- [ ] `between`
- [ ] `regular_points`
- [ ] `clip_string` 
- [ ] `format_array`
- [ ] `new_axis`

Reply:
None of these seem relevant to laziness, most are just doing number operations. I have added "LAZY" notes for sake of consistency, but am undecided whether if it's the right decision.